src/integration_tests/schemas/test_integration_unhappy_path.py

```diff
@@ -0,0 +1,170 @@
+from unittest import TestCase
+import pytest
+from src.app.config.config_factory import config
+from src.integration_tests.helpers.integration_helpers import (
+    cleanup,
+    generate_headers,
+    pubsub_setup,
+    pubsub_teardown,
+    setup_session,
+    inject_wait_time,
+)
+from src.integration_tests.helpers.pubsub_helper import schema_pubsub_helper
+from src.test_data.schema_test_data import test_survey_id
+from src.test_data.shared_test_data import test_schema_subscriber_id
+
+class E2ESchemaIntegrationUnhappyPaths(TestCase):
```

Comment:
Could you move the tests into the schema happy path class and then name it something like SchemaEndpointsIntegrationTest please (if the class has not been named). the tests are not too complicated that they need to be moved. plus the docstrings make them quite easy to distinguish